src/Components/Artwork/__tests__/FillwidthItem.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render } from "enzyme"
+import React from "react"
+import { FillwidthItem } from "../FillwidthItem"
+
+describe("FillwidthItem", () => {
+  // This scenario _should not_ happen. But it did. Every artwork should have
+  //   an image, but somehow one snuck through in production and broke pages.
+  describe("No image associated with an artwork", () => {
+    it("Doesn't blow up when there is no image associated with an artwork", () => {
+      const artwork = {
+        " $fragmentRefs": null,
+        href: "my/artwork",
+      }
+
+      // @ts-ignore
```

Comment:
`ts-ignore` because our types prevent us from passing an artwork that doesn't have an image....in code. 😕

Reply:
😚👌